src/components/molecules/UserProfilePicture/UserProfilePicture.styles.ts

```diff
@@ -34,7 +34,7 @@ export const Reaction = styled.div<ReactionProps>`
   ${({ reactionPosition }) =>
     reactionPosition === "right" ? reactionRight : reactionLeft};
   top: -25px;
-  z-index: 1000;
+  z-index: 10;
```

Comment:
Add this to the z-access helper

Reply:
This is a styled component file

Reply:
Ah, missed that. :( The sooner we can remove those the better IMO

Reply:
> Ah, missed that. :( The sooner we can remove those the better IMO


Agreed